src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -177,17 +179,39 @@ function  azure_rdma_preparation()
 azure_rdma_preparation
 {{/ azRDMA }}
 {{/ reqAzRDMA }}
-# Write env to system-wide environment
-env | grep -E "^PAI|PATH|PREFIX|JAVA|HADOOP|NVIDIA|CUDA" > /etc/environment
+
+function write_user_command()
+{
```

Comment:
We do not need to support multiple line commands here:
* For current version, users can set `command` to `command1 && command2 && ...` or `command1 ; command2 ; ...` to execute multiple commands. If you really think "\n" is useful in `command`, you can add a wrapper to replace `\n` with `;` for `taskData.command` in [mustache renderer](https://github.com/Microsoft/pai/blob/master/src/rest-server/src/models/job.js#L442).
* For future job configuration file, we do support multiple line commands with pre-commands/post-commands, you can refer to [protocol docs](https://github.com/Microsoft/pai/blob/feature/submission-v2/docs/pai-job-protocol.yaml#L183-L196) for example.

Reply:
Thanks for the comment. Yes, complex commands can be chained by `&&` and others, but the use of `\n` makes them much more readable - this is the main reason for this change. Could you name any downsides of multiline commands? It looks like PAI tries to avoid them, is there any reason for that?

Reply:
`\n` is ok but I prefer to support it when rendering mustache (one or two line js code) instead of runtime shell scripts (20+ lines sh code) which will introduce more complexity.

`\n` does make the command more readable, but it's on web page or config file. How we support it inside Docker container wouldn't affect user experience.

Reply:
Doing this during the mustache processing is a good idea and I'm, actually, stand for it. To do this, I think, we have to remove command escaping and directly insert the user command to the target script, possibly forwarding its path. I have no idea how to do this, could you give me some pointers? 

Simple replacement of all the `\n`s should be wrong as they can be a part of the user command.

Reply:
Hi, sorry for the late response.
Replacing all `\n` is not a good idea indeed, but writing command to a target script then execute is not good neither. User may rely on relative path (e.g. `python ./code/run.py`) inside Docker so writing to another path will break existing jobs.

Reply:
Hm, relative paths should be unaffected as long as we don't change current directory. BTW, I thought you are going to deprecate `PAI_CODE_DIR`, aren't you?

Reply:
You're right.
Yes, we are going to deprecate `PAI_CODE_DIR` and let users download code/data themselves. To be more specific, it will be implemented in [`deployments` section](https://github.com/Microsoft/pai/blob/feature/submission-v2/docs/pai-job-protocol.yaml#L198-L220) in the new config file.